optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/test/java/org/optaplanner/quarkus/OptaPlannerProcessorOverridePropertiesAtRuntimeTest.java

```diff
@@ -46,12 +46,23 @@
 
 public class OptaPlannerProcessorOverridePropertiesAtRuntimeTest {
 
+    private static final String QUARKUS_VERSION = getRequiredProperty("version.io.quarkus");
+    private static final String PROJECT_VERSION = getRequiredProperty("project.version");
+
+    private static String getRequiredProperty(String name) {
+        final String v = System.getProperty(name);
+        if (v == null || v.isEmpty()) {
+            throw new IllegalStateException("System property " + name + " has not been set");
+        }
+        return v;
+    }
+
     @RegisterExtension
     static final QuarkusProdModeTest config = new QuarkusProdModeTest()
             .setForcedDependencies(Arrays.asList(
                     // TODO: Remove optaplanner-test when https://github.com/kiegroup/optaplanner/pull/1302 is merged?
```

Comment:
@Christopher-Chianelli Looks like 1302 is merged (through another Pr).
 After this PR is merged - in a separate PR - can you check if we still need optaplanner-test in this list? Either remove it or remove/adjust the comment (as that comment is stale now)

Reply:
It did fail w/o it when I was testing it.